src/statmodels.jl

```diff
@@ -162,6 +162,18 @@ predict(obj::RegressionModel) = error("predict is not defined for $(typeof(obj))
 predict!(obj::RegressionModel) = error("predict! is not defined for $(typeof(obj)).")
 dof_residual(obj::RegressionModel) = error("dof_residual is not defined for $(typeof(obj)).")
 
+"""
+    params(model)
+
+Return all parameters of a model.  For a linear model, this would return coefficients as
+well as any other parameters.
+"""
+params(obj) = error("params is not defined for $(typeof(obj))")
+
+"""
+Write the parameters of a model into preallocated storage.
```

Comment:
Docstrings are required to build the Documenter manual in https://github.com/JuliaStats/StatsBase.jl/pull/275, at least if we want to include the function in the manual. Why wouldn't we export this function? Isn't that the whole point of this issue?